drivers/src/ecc384.rs

```diff
@@ -109,6 +109,11 @@ impl Ecc384PubKey {
     pub fn to_der(&self) -> [u8; 97] {
         array_concat3([0x04], (&self.x).into(), (&self.y).into())
     }
+
+    pub fn zeroize(&mut self) {
```

Comment:
Have you looked at the machine code to see if this actually happens? I would expect the optimizer to remove this.